components/Buttons/src/MDCButton.m

```diff
@@ -252,10 +262,13 @@ - (void)layoutSubviews {
 
     CGFloat offsetX = contentCenterPoint.x - boundsCenterPoint.x;
     CGFloat offsetY = contentCenterPoint.y - boundsCenterPoint.y;
-    _inkView.frame =
+    CGRect inkFrame =
         CGRectMake(offsetX, offsetY, CGRectGetWidth(self.bounds), CGRectGetHeight(self.bounds));
+    _inkView.frame = inkFrame;
+    self.rippleView.frame = inkFrame;
   } else {
-    _inkView.frame = self.bounds;
+    _inkView.frame = CGRectStandardize(self.bounds);
```

Comment:
can you standardize in the initialization above so you don't issue CGRectStandarddize twice?

Reply:
Done